Library/Homebrew/livecheck.rb

```diff
@@ -24,13 +29,10 @@ class Livecheck
   sig { returns(T.nilable(Proc)) }
   attr_reader :strategy_block
 
-  # Options used by `Strategy` methods to modify `curl` behavior.
-  sig { returns(T.nilable(T::Hash[Symbol, T.untyped])) }
-  attr_reader :url_options
-
   sig { params(package_or_resource: T.any(Cask::Cask, T.class_of(Formula), Resource)).void }
   def initialize(package_or_resource)
     @package_or_resource = package_or_resource
+    @options = T.let(Homebrew::Livecheck::Options.new, Homebrew::Livecheck::Options)
```

Comment:
This is minor, but if you bump it up a line, the attrs will remain in alpha order

Reply:
Though these were strictly in alphabetical order before, that was only a coincidence. These are arranged as "arguments (`package_or_resource`) first, then everything else in alphabetical order", which is maintained here.